docs/content/installation/upgrade.md

```diff
@@ -13,5 +13,6 @@ Use this information to upgrade the IBM® block storage CSI driver.
 
 - To manually upgrade the CSI (Container Storage Interface) driver from a previous version with GitHub, perform step 1 of the [installation procedure](install_driver_github.md) for the latest version.
 
-
-
+**Important:** When upgrading from IBM® block storage CSI driver 1.9.0 or below, the host definer must be manually installed. Once the the host definer is initially installed, the host definer automatically updates along with the driver version update during seamless upgrade. 
```

Comment:
>the host definer must be manually installed

this might imply that installing the host definer is not optional, when in fact it **is** optional

Reply:
updated